app/models/brand.rb

```diff
@@ -182,8 +184,47 @@ def in_trial?
     !!subscription.try(:trial?)
   end
 
+  def delete_account
```

Comment:
Right now this is going to be something that an only an admin person can perform. We really want to limit this only to people that _really_ want to delete their account. For all others, I'm fine with just letting people pause/cancel their subscription. The actually closing of accounts will be something that we make people email us about. In that scenario, I'm fine with soft deleting _everything_ including the brand's identities. That way _if_ they really, really, really want to use the service again, they will start fresh. I like this approach because then we don't have to worry about resurrecting accounts. To do this, we'll need to make [this](https://github.com/signalplus-dev/signalplus/blob/master/db/schema.rb#L41) be unique on the compound index of `["provider", "uid", "deleted_at"]` and add a `deleted_at` column to the `identities` table.